lib/ui/views/installer/installer_view.dart

```diff
@@ -24,7 +24,7 @@ class InstallerView extends StatelessWidget {
             floatingActionButton: Visibility(
               visible: !model.isPatching && !model.hasErrors,
               child: model.isInstalled
-                  ? FloatingActionButton.extended(
+                  ? HapticFloatingActionButtonExtended(
```

Comment:
Changed to `HapticFloatingActionButtonExtended` to resolve merge conflict with `dev` branch.